style.css

```diff
@@ -1,34 +1,204 @@
 * {
   margin: 0;
   padding: 0;
+}
+:root {
+  --home-bg-color: #99dfff;
+  --home-primary-color: #43c4ff;
+  --home-secondary-color: #0095da;
+  --black-color: #000000;
+  --white-color: #ffffff;
+}
+
+/* body {
+  background-image: linear-gradient(
+    140deg,
+    rgb(255, 255, 255),
+    var(--home-bg-color)
+  );
+  background-repeat: no-repeat;
+} */
+
+.main-container {
+  height: 100vh;
+  width: 100%;
+  padding: 0 5%;
```

Comment:
A suggestion here would be that you could use `rem` as a unit for padding because at all other places you've used `rem` as a unit. It helps in maintaining consistency.